src/Acuminator/Acuminator.Analyzers/Analyzers/Localization/LocalizationPXExceptionAnalyzer.cs

```diff
@@ -84,17 +85,18 @@ private ExpressionSyntax GetMessageExpression(SyntaxNodeAnalysisContext syntaxCo
 
         private void AnalyzeThrowOfPXException(SyntaxNodeAnalysisContext syntaxContext, PXContext pxContext)
```

Comment:
@metallmonster, since the analysis is done for constructors now I think that method should change its name

Reply:
@SENya1990 yes